easybuild/easyblocks/b/boost.py

```diff
@@ -252,7 +252,10 @@ def sanity_check_step(self):
             pymajorver = get_software_version('Python').split('.')[0]
             pyminorver = get_software_version('Python').split('.')[1]
             if int(pymajorver) >= 3:
-                suffix = pymajorver
+                if LooseVersion(self.version) >= LooseVersion("1.68.0"):
+                    suffix = '%s%s' % (pymajorver, pyminorver)
+                else:
+                    suffix = pymajorver
```

Comment:
I didn't, and I suspect @rubendibattista missed it too...

Reply:
I was worried about backwards compatibility since I did not know if it was just for new stuff or also for some previous versions and I did not tried with Python 3. So I didn't want to mess up with things I did not test. 

I think the proposed solution by @boegel looks good. :)

Sorry about the overlook.

Reply:
@rubendibattista @boegel 

Didn't mean to upset and hope I didn't.  When I saw the code I assumed there was a reason for it looking as it did.  I submitted a revised version (currently testing).